### PR TITLE
adding support for different transport_method

### PR DESCRIPTION
**Pull request by sherl0cks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,9 +11,22 @@ This repository provides a set of generic roles and reusable playbooks for deplo
 3. Clone the [JBoss EAP Role](https://github.com/rhtconsulting/jboss_eap) into your [Ansible Roles Path](http://docs.ansible.com/ansible/intro_configuration.html#roles-path). Future version will support galaxy to make this easier.
 4. TODO: ansible-galaxy install
 
+Transfer Methods
+----------
+
+These playbooks and their required roles support a few different mechanisms for transferring the product zip files to the target host.
+
+### csp-to-host
+This method uses the [custom redhat_csp_download module](https://github.com/rhtconsulting/ansible-middleware-playbooks/blob/master/library/redhat_csp_download.py) to download the product binaries from the [Red Hat Customer Portal](https://access.redhat.com/downloads/). This requires network access from the target host to the Red Hat Customer Portal, but has the advantage of being fully automated. Each of the roles have sensible defaults already set.
```
Comment (sabre1041) [inline README.md]: Can the reference to the `redhat_csp_download.py` file be a relative path instead of the GitHub URL

**Comment by sabre1041** [comment]
@sherl0cks minor updates needed otherwise we are looking good. Only other thing that came to mind is the inventory file and if we wanted to provide comments/helpful hints. Thoughts?

**Comment by sherl0cks** [comment]
@sabre1041 fixed your relative path thing. RE: inventory file, I really don't have thoughts there. It's pretty straight forward. I'd rather get people using it an provide feedback than add a ramble of comments no one needs

**Comment by sabre1041** [comment]
@sherl0cks works for me. Lets take what we have now

**Pull request merged**